src/pipecat/transports/network/fastapi_websocket.py

```diff
@@ -114,37 +122,66 @@ async def write_raw_audio_frames(self, frames: bytes):
                 wav_frame = AudioRawFrame(
                     content.read(),
                     sample_rate=frame.sample_rate,
-                    num_channels=frame.num_channels)
+                    num_channels=frame.num_channels,
+                )
                 frame = wav_frame
 
             payload = self._params.serializer.serialize(frame)
             if payload and self._websocket.client_state == WebSocketState.CONNECTED:
                 await self._websocket.send_text(payload)
 
-            self._websocket_audio_buffer = self._websocket_audio_buffer[self._params.audio_frame_size:]
+            self._websocket_audio_buffer = self._websocket_audio_buffer[
+                self._params.audio_frame_size:
+            ]
+
+    async def write_websocket_clear_event(self, frame: StartInterruptionFrame):
+        payload = self._params.serializer.serialize(frame)
+        if payload and self._websocket.client_state == WebSocketState.CONNECTED:
+            await self._websocket.send_text(payload)
```

Comment:
It seems this is not used and probably not necessary. This could be achieved from any processor by sending a `StartInterruptionFrame`. In general, if we want to achieve something in Pipecat we do it through frames, instead of calling functions directly.

Reply:
You are totally right. I forgot this class was based on `BaseOutputTransport` (it originally wasn't at some point).